packages/amplify_flutter/lib/amplify.dart

```diff
@@ -173,10 +163,10 @@ class AmplifyClass extends PlatformInterface {
     }
 
     try {
-      var res = await AmplifyClass.instance
+      bool? res = await AmplifyClass.instance
           ._configurePlatforms(_getVersion(), configuration);
-      _isConfigured = res;
-      if (!res) {
+      _isConfigured = res!;
```

Comment:
Isn't this the equivalent of saying `res` might be null and then immediately casting it as non-nullable without checking if it was in fact non-null? Should this instead be something like:
```dart
_isConfigured = res ?? false;
```

Reply:
good point, you're right, updated the code with your suggestion.